zio-http/src/main/scala/zio/http/netty/NettyBodyWriter.scala

```diff
@@ -0,0 +1,72 @@
+package zio.http.netty
+
+import io.netty.buffer.Unpooled
+import io.netty.channel._
+import io.netty.handler.codec.http.LastHttpContent
+import zio._
+import zio.http.Body
+import zio.http.Body._
+
+object NettyBodyWriter {
+
+  def write(body: Body, ctx: ChannelHandlerContext): Task[Boolean] = body match {
+    case body: ByteBufBody            =>
+      ZIO.attempt(ctx.write(body.byteBufData): Unit).as(false)
```

Comment:
Much faster to do:
```suggestion
      ZIO.attempt { ctx.write(body.byteBufData); false })
```

Fewer ZIO ops = faster.